3drepo/3drepo.io
# ISSUE #3317 - SafetiBase - Risk details

This fixes #3317 

#### Description
- fixes the styling in the V5 viewer for the SafetiBase Risk tab

#### Test cases
- View existing SafetiBase risks
- Create a new SafetiBase risk and check the UI isn't whack.
